driver-core/src/test/unit/com/mongodb/client/model/FindOneAndUpdateOptionsSpecification.groovy

```diff
@@ -115,4 +115,21 @@ class FindOneAndUpdateOptionsSpecification extends Specification {
         where:
         arrayFilters << [null, [], [new BsonDocument('a.b', new BsonInt32(1))]]
     }
+
+
+    def 'should set hint'() {
```

Comment:
Were there tests just missing from a previous change?

Reply:
I have removed the tests since they are duplicates of the tests in UpdateOptionsSpecification.

Reply:
I don't they should be duplicates, since they should actually use FindOneAndUpdateOptions not UpdateOptions.  But let's fix that in a separate commit.